### PR TITLE
Updated pyshp to 1.2.3

### PR DESCRIPTION
**Pull request by ocefpaf**
The AppVeyor failures will go away with #112

### CODE REVIEW

File: pyshp/build.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-$PYTHON setup.py install
+$PYTHON setup.py install install --single-version-externally-managed --record record.txt
```
Comment (pelson) [inline pyshp/build.sh]: I think we might be able to start pulling these one liners (which are shared between all platforms) into the meta.yaml.
What do you think?

File: pyshp/bld.bat [hunk 2]
```diff
@@ -1 +1,2 @@
-%PYTHON% setup.py install
\ No newline at end of file
+%PYTHON% setup.py install install --single-version-externally-managed --record record.txt
```
Comment (rhattersley) [inline pyshp/bld.bat]: Is this supposed to say "install" twice?

Comment (ocefpaf) [inline pyshp/bld.bat]: Nope.  This say "Filipe stop sending PRs at the end of the day."

File: pyshp/build.sh (hunk 1 above)
Comment (ocefpaf) [inline pyshp/build.sh]: Big :+1:

File: pyshp/bld.bat (hunk 2 above)
Comment (rhattersley) [inline pyshp/bld.bat]: Aren't you GMT-6? So... it's 3 in the morning?! :scream:

Comment (ocefpaf) [inline pyshp/bld.bat]: Good no!  My adolescent days of late night coding are over!! 

I am GMT-3 (Northeast Brazil).  GitHub says that commit was ~17:30 and that is late for my age :wink:

**Pull request merged**